packages/syft/src/syft/client/client.py

```diff
@@ -477,6 +477,16 @@ def __init__(
 
         self.post_init()
 
+    def rrr(self):
```

Comment:
@IonesioJunior Was this committed by accident?

Reply:
ah that was me, will remove

Reply:
added it while debugging the reconstruction of the stacktrace for syft functions, with a pointer to the line that throws the error